### PR TITLE
Collate data for retraining pipelines

### PR DESCRIPTION
**Pull request by miteshvp**

### CODE REVIEW

File: f8a_report/report_helper.py [hunk 1]
```diff
@@ -159,16 +189,86 @@ def normalize_deps_list(self, deps):
...
+            if bucket_name:
+                logger.info('Storing user-input stacks for ecosystem {eco} at {dir}'.format(
+                    eco=eco, dir=bucket_name + obj_key))
+                self.s3.store_json_content(content=training_data, bucket_name=bucket_name,
+                                           obj_key=obj_key)
+
     def normalize_worker_data(self, start_date, end_date, stack_data, worker, frequency='weekly'):
         """Normalize worker data for reporting."""
         total_stack_requests = {'all': 0, 'npm': 0, 'maven': 0}
+        report_name = dt.strptime(end_date, '%Y-%m-%d').strftime('%Y-%m-%d')
```
Comment (samuzzal-choudhury) [inline f8a_report/report_helper.py]: @miteshvp  Does this mean that even the monthly report names are going to be '%Y-%m-%d'?

File: tests/test_stack_report_helper.py [hunk 2]
```diff
@@ -174,30 +182,66 @@ def test_populate_key_count_failure():
         assert e.value == 'TypeError("unhashable type: \'list\'",)'
 
 
-def test_S3helper():
+def test_s3helper():
```
Comment (samuzzal-choudhury) [inline tests/test_stack_report_helper.py]: @miteshvp is this change required?

File: f8a_report/report_helper.py (hunk 1 above)
Comment (miteshvp) [reply f8a_report/report_helper.py]: Missed it. Added now

File: tests/test_stack_report_helper.py (hunk 2 above)
Comment (miteshvp) [reply tests/test_stack_report_helper.py]: Based on the pythonic way, definitions are lowercase
